pheps/phep-0001.md

```diff
@@ -0,0 +1,338 @@
+
+PHEP authors are responsible for managing the progress of the discussion of their PHEP, for instance by updating the PR description to highlight topics of particular interest and to provide the dates of votes on the PHEP resolution.
+
+If a PHEP undergoes a significant re-write or other major changes, the author(s) and editor may discuss closing the existing PR and opening a new one, starting from the changed version. If this occurs, the `Discussions-To` link must be updated.
+
+PHEP discussions are subject to the [PyHC Code of Conduct](https://heliopython.org/docs/code_of_conduct/).
+
+## PHEP Review & Resolution
+<a name="phep-review-resolution"></a>
+Once the authors have completed a PHEP, they may request a review for style and consistency from the PHEP editors. However, content review and acceptance of the PHEP is the responsibility of the community. A PHEP is accepted by consensus of attendees (virtual or in-person) at both a PyHC regular telecon and a regular PyHC meeting, with no edits to the PHEP occurring in between. Any edits "reset the clock" and two further votes must be taken. Potential acceptance of a PHEP must be placed on the agenda at least a month before the meeting, including a link to the open PR with the proposed-final language of the PHEP. Persons not in attendance at the meetings may object via comment on the PR.
+
+A "regular PyHC meeting" is any meeting (virtual, physical, or hybrid) organized for the purpose of discussing PyHC business, of at least a half-day duration, where a substantial portion of the community is expected to participate, and announced as such well in advance. This includes the semiannual meetings but may also include side gatherings at other meetings which are of interest to the community and designated as PyHC meetings.
+
+"Consensus" here means no concrete objections have been raised: an accepted PHEP must be one that the entire community can at least live with. The PHEP authors must make the case for the PHEP; objectors must provide meaningful concerns that could, in principle, be addressed by the authors. All are expected to discuss in good faith, while recognizing that good faith does not guarantee consensus can be reached.
+
+For a PHEP to be accepted it must meet certain minimum criteria. It must be a clear and complete description of the proposed change. The change must represent a net improvement. The proposed implementation, if applicable, must be solid and must not complicate PyHC projects unduly. Finally, a proposed enhancement must be "pythonic" in order to be accepted by the community. ("Pythonic" is an imprecise term; it may be defined as whatever is acceptable to the community. This logic is intentionally circular.)
```

Comment:
Can we clarify "solid"? Again, I realize that the same language appears in PEP 1.

I also think it would be nice to somehow tie our use of "pythonic" to the Python community's (and Steering Committee's) use of the term. It is unlikely that the PyHC notion of what is "pythonic" should significantly differ from the broader Python community's notion of what is "pythonic", but maybe we can formalize that connection a bit.

Reply:
I'm relucant to define "solid" as this is a very wibbly section where it's hard to have the appropriate degree of vagueness. I'm not sure if there's "case law" in how this has been interpreted for PEP-1. Maybe another adjective would be better.

Agreed the connection of "Pythonic" should be beyond PyHC, try:
("Pythonic" is an imprecise term; ~it may be defined as whatever is acceptable to the community. This logic is intentionally circular~it should be understood in the context of the broad Python ecosystem.)

Reply:
[New wording re:Pythonic](https://github.com/jtniehof/standards/blob/e335aacd1346f0876f5a54d9d617bf3c11ec2e32/pheps/phep-0001.md?plain=1#L129)